fusor-ember-cli/app/templates/review/installation.hbs

```diff
@@ -265,53 +265,37 @@
                               routeName='subscriptions.credentials'
                               isRequired=true
                               value=model.manifest_file }}
-
-                {{#each reviewSubscriptions as |sub|}}
-                    {{review-link label='Subscription Name'
-                                  routeName='subscriptions.select-subscriptions'
-                                  value=sub.product_name }}
-                    {{review-link label='Contract Number'
-                                  routeName='subscriptions.select-subscriptions'
-                                  value=sub.contract_number }}
-                    {{review-link label='Quantity'
-                                  routeName='subscriptions.select-subscriptions'
-                                  value=sub.quantity_attached }}
-                {{else}}
-                      {{review-link label='Quantity'
-                                    routeName='subscriptions.select-subscriptions'
-                                    value="0 - no subscriptions in manifest" }}
-                {{/each}}
-
-            {{/accordion-item}}
-
-        {{else}}
-            {{#accordion-item name='Subscriptions' isOpen=isSubscriptionsOpen}}
+            {{else}}
                 {{review-link label='Subscription Management Application'
                               routeName='subscriptions.management-application' isRequired=true
                               value=deploymentController.managementApplicationName }}
-
-                {{#each reviewSubscriptions as |sub|}}
-                    {{review-link label='Subscription Name'
-                                  routeName='subscriptions.select-subscriptions'
-                                  value=sub.product_name }}
-                    {{review-link label='Contract Number'
-                                  routeName='subscriptions.select-subscriptions'
-                                  value=sub.contract_number }}
-                    {{review-link label=qtyLabel
+            {{/if}}
+
+            {{#each reviewSubscriptions as |sub|}}
+                {{review-link label='Subscription Name'
+                              routeName='subscriptions.select-subscriptions'
+                              value=sub.product_name }}
+                {{review-link label='Contract Number'
+                              routeName='subscriptions.select-subscriptions'
+                              value=sub.contract_number }}
+                {{review-link label='Quantity'
```

Comment:
Thanks. When looking at the review page, it wasn't clear by the label just
as "Quantity"
On May 23, 2016 7:41 PM, "jesus m. rodriguez" <notifications@github.com>
wrote:

> In fusor-ember-cli/app/templates/review/installation.hbs
> <https://github.com/fusor/fusor/pull/873#discussion_r64248821>:
>
> > -                                  routeName='subscriptions.select-subscriptions'
> > -                                  value=sub.product_name }}
> > -                    {{review-link label='Contract Number'
> > -                                  routeName='subscriptions.select-subscriptions'
> > -                                  value=sub.contract_number }}
> > -                    {{review-link label=qtyLabel
> > +            {{/if}}
> > +
> > +            {{#each reviewSubscriptions as |sub|}}
> > +                {{review-link label='Subscription Name'
> > +                              routeName='subscriptions.select-subscriptions'
> > +                              value=sub.product_name }}
> > +                {{review-link label='Contract Number'
> > +                              routeName='subscriptions.select-subscriptions'
> > +                              value=sub.contract_number }}
> > +                {{review-link label='Quantity'
>
> Sounds reasonable. I will make that change.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/fusor/fusor/pull/873/files/e5d692538711a09644c6d7c6d51174c2c6a1a67e#r64248821>
>